packages/gatekeeper/src/components/TokenExpiry/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Col, Container, Row } from 'reactstrap';
+import { LOGIN_BUTTON_TEXT, SESSION_EXPIRED_TEXT } from './constants';
+
+/** TokenExpired props interface */
+export interface TokenExpiredProps {
+  logoutLinkText: string; // text on the logout link button
+  logoutUrl: string;
+  sessionExpiryText: string; // text informing users their session has expired
+}
+
+/** default page to display when token expires */
+const TokenExpired = (props: TokenExpiredProps) => {
```

Comment:
Where is this component used?

Reply:
will act as a default component which user can be redirected to when a token is expired.

Reply:
Should we add that to the docs of gatekeeper?